data/hashstack.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Hashstack",
+  "description": "Hashstack provides a permissionless under-collateralised lending solution for crypto-native retail, enabling them to borrow up to 300% of the collateral. Product status: Public testnet on Starknet.",
+  "short_description": "Permissionless undercollateralized loans",
+  "tags": [
+    "DeFi"
+  ], 
+  "contracts": [],
+  "goerliContracts": [
+    {
+      "name": "Diamond Contract",
+      "address": "https://testnet-2.starkscan.co/contract/0x05ad65a92e83d652175aa914e922afd2917f6a9520047eaa0da0684749163fc4"
```

Comment:
@web3Gurung please update this to the contract addresses only

Reply:
I will make the required changes, thank you :)